### DISCUSSION TITLE
ejs File support

### DISCUSSION THREAD

**Discussion by ChristianPerez34** [zed-industries/zed#15130] [Feature Requests]
Hi all, 

I've recently been working on a project that uses ejs. As of now there is no syntax highlighting or intellisense for these files. I can manually set it to use Javascript as the lsp, however I would like to see if there is some configuration I can set so that it is automatically treated as a Javascript file.

**Comment by ChristianPerez34** [comment]
Noticed there is a file_types setting that can be used to associate files. While I initially said Javascript I've made associations to HTML as it seemed to work better

```json
{
  "file_types": {
    "HTML": ["ejs"]
  }
}
```

**Reply by willsrockson** [reply]
There is an Extention for Ejs now, but none of the auto-suggestions work. i hope EJS get fully supported soon

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
